exporters/otlp/BUILD

```diff
@@ -28,3 +28,12 @@ cc_library(
         "@com_github_opentelemetry_proto//:trace_proto_cc",
     ],
 )
+
+cc_test(
```

Comment:
Please add the test also to the `CMakeList.txt` in this directory, similar to [this](https://github.com/open-telemetry/opentelemetry-cpp/blob/master/sdk/test/trace/CMakeLists.txt#L6).